samples/crypto/test_sample/README.rst

```diff
@@ -0,0 +1,258 @@
+| Hash               | SHA256      | `NIST - SHA`_                                                              | `NIST - SHA`_                                                              |
++--------------------+-------------+----------------------------------------------------------------------------+----------------------------------------------------------------------------+
+|                    | SHA512      |                                                                            | `NIST - SHA`_                                                              |
++--------------------+-------------+----------------------------------------------------------------------------+----------------------------------------------------------------------------+
+| HMAC               | HMAC SHA256 | `NIST - HMAC`_                                                             | `NIST - HMAC`_                                                             |
++--------------------+-------------+----------------------------------------------------------------------------+----------------------------------------------------------------------------+
+|                    | HMAC SHA512 |                                                                            | `NIST - HMAC`_                                                             |
++--------------------+-------------+----------------------------------------------------------------------------+----------------------------------------------------------------------------+
+| HKDF               | HKDF SHA256 | `RFC 5869 - HMAC-based Extract-and-Expand Key Derivation Function (HKDF)`_ | `RFC 5869 - HMAC-based Extract-and-Expand Key Derivation Function (HKDF)`_ |
++--------------------+-------------+----------------------------------------------------------------------------+----------------------------------------------------------------------------+
+|                    | HKDF SHA512 |                                                                            | `RFC 5869 - HMAC-based Extract-and-Expand Key Derivation Function (HKDF)`_ |
++--------------------+-------------+----------------------------------------------------------------------------+----------------------------------------------------------------------------+
+
+
+Requirements
+************
+
+* One of the following development boards:
+
+  * |nRF9160DK|
+  * |nRF5340DK|
+  * |nRF52840DK|
+  * |nRF52DK|
+  * |nRF51DK|
+
+* something else??
+
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/bluetooth/central_bas`
+
+.. include:: /includes/build_and_run.txt
+
+.. _crypto_test_testing:
+
+Testing
+=======
+
+1. Compile and program the application.
+#. Observe the result of the different test vectors in the log using an RTT viewer or a terminal emulator.
+   Make sure the test vector verdict is "passed"::
```

Comment:
The output is different in ztest. Only one single line is reported at the end

Successful run:
PROJECT EXECUTION SUCCESSFUL

Failed run: 
PROJECT EXECUTION FAILED

Reply:
done